amethyst_animation/src/lib.rs

```diff
@@ -1,3 +1,50 @@
+//! Provides computer graphics animation functionality.
+//!
+//! Animation on a single entity comprises of one or more [`Sampler`][sampler]s. Each sampler
+//! operates on a [`Channel`][channel]. Thus, for a single entity, conceptually each
+//! `(Channel, Sampler)` pair is enough to define one part the animation, and a
+//! `Vec<(Channel, Sampler)>` defines the whole animation.
+//!
+//! In a more complex situation, an object in game may be made up of multiple entities. Say you have
+//! a dragon monster, that is defined by a skinned mesh that has a skeleton with 10 joints. Each
+//! joint will then be an `Entity`. Our animation definition holds the samplers to run for the whole
+//! object. To animate each of the entities of this complex object, we need a way to link the
+//! sampler to the each of the entities.
+//!
+//! Animation definitions are persistent and can be stored on disk. Entities however, are not. To
```

Comment:
Entities are not, but they will soon

Reply:
No, the Entity itself cannot be stored. Markers can.